dev/CommonStyles/Common_themeresources.xaml

```diff
@@ -18,8 +18,8 @@
             <Thickness x:Key="TextControlThemePadding">10,5,6,6</Thickness>
             <SolidColorBrush x:Key="ApplicationPageBackgroundThemeBrush" Color="{StaticResource SolidBackgroundFillColorBase}" />
             <StaticResource x:Key="DefaultTextForegroundThemeBrush" ResourceKey="TextFillColorPrimaryBrush" />
-            <SolidColorBrush x:Key="SystemControlFocusVisualPrimaryBrush" Color="{StaticResource FocusStrokeColorOuter}"/>
-            <SolidColorBrush x:Key="SystemControlFocusVisualSecondaryBrush" Color="{StaticResource FocusStrokeColorInner}"/>
+            <StaticResource x:Key="SystemControlFocusVisualPrimaryBrush" ResourceKey="FocusStrokeColorOuter"/>
+            <StaticResource x:Key="SystemControlFocusVisualSecondaryBrush" ResourceKey="FocusStrokeColorInner"/>
```

Comment:
@marksfoster FYI

Reply:
These should point to the brush resources: ResourceKey="FocusStrokeColorOuterBrush"

Reply:
Fixed now, thank you @marksfoster for pointing that out.